devolutions-session/src/dvc.rs

```diff
@@ -15,8 +15,8 @@ use windows::Win32::System::IO::{GetOverlappedResult, OVERLAPPED};
 
 const CHANNEL_PDU_LENGTH: usize = 1024;
 
-pub fn loop_dvc(config: ConfHandle) {
-    if !config.get_conf().debug.enable_unstable {
+pub fn loop_dvc(conf: &Conf) {
+    if !conf.debug.enable_unstable {
         debug!("DVC loop is disabled");
         return;
     }
```

Comment:
suggestion: I think it would be better to make the check for `enable_unstable` outside of this function. As it appears, this function does not really require the `Conf` object.

This is better style:
```rust
    if config.get_conf().debug.enable_unstable { // or conf.debug.enable_unstable if you have that at hand
        loop_dvc(config);
    }
```

Here is also a read I recommend: https://matklad.github.io/2023/11/15/push-ifs-up-and-fors-down.html

Reply:
cc @pacmancoder

Reply:
Addressed in [54772a4](https://github.com/Devolutions/devolutions-gateway/pull/1068/commits/54772a44f1f704cc22b2aee6d56e28bd82fb4b70).